src/raftstore/store/worker/apply.rs

```diff
@@ -832,7 +877,10 @@ impl ApplyDelegate {
         ctx: &mut ApplyContext,
     ) -> Result<(RaftCmdResponse, Option<ExecResult>)> {
         let req = Rc::clone(&ctx.exec_ctx.as_ref().unwrap().req);
-        check_epoch(&self.region, &req)?;
+        // Include region for stale epoch after merge may cause key not in range.
+        let include_region =
+            req.get_header().get_region_epoch().get_version() >= self.last_merge_version;
```

Comment:
need to check last_merge_version 0 here?

Reply:
If `last_merge_version` is 0, `include_region` becomes true, which is expected.